tests/rgen/Microsoft.Macios.Generator.Tests/Classes/Data/Trampolines/ExpectedAVFoundationTrampolinesTrampolines.cs

```diff
@@ -0,0 +1,1369 @@
+				return default (global::ObjCRuntime.BlockLiteral);
+			return CreateBlock (callback);
+		}
+
+		[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+		internal static unsafe global::ObjCRuntime.BlockLiteral CreateBlock (global::AVFoundation.AVAudioNodeTapBlock callback)
+		{
+			delegate* unmanaged<global::System.IntPtr, global::ObjCRuntime.NativeHandle, global::ObjCRuntime.NativeHandle, void> trampoline = &Invoke;
+			return new global::ObjCRuntime.BlockLiteral (trampoline, callback, typeof (SDAVAudioNodeTapBlock), nameof (Invoke));
+		}
+	}
+	// TODO: generate trampoline for AVFoundation.AVAudioNodeTapBlock
+
+	[UnmanagedFunctionPointerAttribute (CallingConvention.Cdecl)]
+	[UserDelegateType (typeof (global::AVFoundation.AVAudioSequencerUserCallback))]
+	unsafe internal delegate void DAVAudioSequencerUserCallback (global::System.IntPtr block_ptr, global::ObjCRuntime.NativeHandle track, global::ObjCRuntime.NativeHandle userData, double timeStamp);
+
+	/// <summary>This class bridges native block invocations that call into C#</summary>
+	static internal class SDAVAudioSequencerUserCallback
+	{
+		[Preserve (Conditional = true)]
+		[UnmanagedCallersOnly]
+		[UserDelegateType (typeof (global::AVFoundation.AVAudioSequencerUserCallback))]
+		internal static unsafe void Invoke (global::System.IntPtr block_ptr, global::ObjCRuntime.NativeHandle track, global::ObjCRuntime.NativeHandle userData, double timeStamp)
+		{
+			var del = global::ObjCRuntime.BlockLiteral.GetTarget<global::AVFoundation.AVAudioSequencerUserCallback> (block_ptr);
+			if (del is not null)
+			{
+				del (global::ObjCRuntime.Runtime.GetNSObject<global::AVFoundation.AVMusicTrack> (track)!, global::ObjCRuntime.Runtime.GetNSObject<global::Foundation.NSData> (userData)!, timeStamp);
+			}
+		}
+
+		internal static unsafe global::ObjCRuntime.BlockLiteral CreateNullableBlock (global::AVFoundation.AVAudioSequencerUserCallback? callback)
+		{
+			if (callback is null)
+				return default (global::ObjCRuntime.BlockLiteral);
+			return CreateBlock (callback);
+		}
+
+		[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+		internal static unsafe global::ObjCRuntime.BlockLiteral CreateBlock (global::AVFoundation.AVAudioSequencerUserCallback callback)
+		{
+			delegate* unmanaged<global::System.IntPtr, global::ObjCRuntime.NativeHandle, global::ObjCRuntime.NativeHandle, double, void> trampoline = &Invoke;
+			return new global::ObjCRuntime.BlockLiteral (trampoline, callback, typeof (SDAVAudioSequencerUserCallback), nameof (Invoke));
+		}
+	}
+	// TODO: generate trampoline for AVFoundation.AVAudioSequencerUserCallback
+
+	[UnmanagedFunctionPointerAttribute (CallingConvention.Cdecl)]
+	[UserDelegateType (typeof (global::AVFoundation.AVAudioSinkNodeReceiverHandlerRaw))]
+	unsafe internal delegate int DAVAudioSinkNodeReceiverHandlerRaw (global::System.IntPtr block_ptr, IntPtr timestamp, uint frameCount, IntPtr inputData);
```

Comment:
There is a bug here, we should have the 'global' alias for the IntPtr. This has been fixed in https://github.com/dotnet/macios/pull/22931